docs/releases/2.0.txt

```diff
@@ -361,6 +365,11 @@ Miscellaneous
   ``PASSWORD_RESET_TIMEOUT_DAYS = 3``, password reset tokens are now valid for
   72 hours rather than 96 hours.
 
+* The default size of the Oracle test tablespace has increased from 20M to 50M.
```

Comment:
Single bullet point seems okay: "The default size of the Oracle test tablespace is increased from 20M to 50M and the default autoextend size is increased ..."